### PR TITLE
Patch amz ord 1

### PR DESCRIPTION
**Pull request by orenlande**
Sorry for late PR, was completely forgot about it.
Referring to the issue at https://github.com/keithbrink/amazon-mws-laravel/issues/31.

Next step would to add some more useful docs to the readme.

### CODE REVIEW

File: src/AmazonOrderList.php [hunk 1]
```diff
@@ -469,15 +470,17 @@ protected function parseXML($xml)
...
+
+            $order = new AmazonOrder(
                 $this->storeName,
                 null,
                 $data,
                 $this->mockMode,
                 $this->mockFiles,
                 $this->config
             );
-            $this->orderList[$this->index]->mockIndex = $this->mockIndex;
```
Comment (keithbrink) [inline src/AmazonOrderList.php:480]: This line needs to stay for tests I believe.

**Review by keithbrink — CHANGES_REQUESTED** [review]

File: src/AmazonOrderList.php (hunk 1 above)
Comment (orenlande) [reply src/AmazonOrderList.php:480]: I was thinking the same. Unfortunately leaving it in as it makes the code fail and request not passing on. Couldn't figure out if where the `$this->mockIndex` is coming from, seems like non-exists property. 

Any thoughts?

Comment (keithbrink) [reply src/AmazonOrderList.php:480]: Well, it's setting the mockIndex on the AmazonOrder object...I'm wondering if a better approach here might be to leave this as is, and you should be able to do something like this:

```
$order_list = new AmazonOrderList(....);
...
$order_list->getList();
foreach($order_list as $order) {
    $order_data = $order->getData();
}
```

Could you try that and let me know if that works?

Comment (keithbrink) [reply src/AmazonOrderList.php:480]: And BTW, `mockIndex` is a property on `AmazonCore`.

Comment (orenlande) [reply src/AmazonOrderList.php:480]: You're right. it's actually working fine when putting it back - I think it's something with config although not really sure about it. anyway, looks like it working

**Comment by keithbrink** [comment]
I'm going to close this PR in favour of getting the data like so:

```
$order_list = new AmazonOrderList(....);
...
$order_list->getList();
foreach($order_list as $order) {
    $order_data = $order->getData();
}
```

**Pull request closed without merging**